phpstan-baseline-standard.neon

```diff
@@ -3670,6 +3570,11 @@ parameters:
 			count: 1
 			path: library/Businessprocess/Storage/LegacyStorage.php
 
+		-
+			message: "#^Method Icinga\\\\Module\\\\Businessprocess\\\\Storage\\\\Storage\\:\\:__construct\\(\\) has parameter \\$config with generic class Icinga\\\\Data\\\\ConfigObject but does not specify its types\\: TValue$#"
```

Comment:
This is related to a recent change in Icinga Web and should be properly fixed.

Reply:
Fixed in https://github.com/Icinga/icingaweb2-module-businessprocess/pull/446